### PR TITLE
Store manifest in FilePublication

### PR DESCRIPTION
**Pull request by lubosmj**
closes #7838

### CODE REVIEW

File: CHANGES/7838.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Fixed a bug which caused the plugin to publish the default manifest's name instead of the specified
+one.
```
Comment (mdellweg) [inline CHANGES/7838.bugfix:2]: Actually, i believe it published the right one, but reported the wrong one on the publication resource.

**Review by mdellweg — COMMENTED** [review]
This looks right to me code wise.
I think the changelog needs clarification though.

File: CHANGES/7838.bugfix [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Fixed a bug which caused the plugin to report the default manifest's name instead of the specified
+one in the publication endpoint.
```
Comment (mdellweg) [inline CHANGES/7838.bugfix:2]: :+1:

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
It does not say it needs a rebase, but *just* that it is outdated. Has anyone configured to only allow fast forward merges?

**Review by dkliban — APPROVED** [review]

**Review by dkliban — APPROVED** [review]

**Pull request merged**